Code/Engine/Foundation/Basics/Platform/Android/AndroidUtils.h

```diff
@@ -11,6 +12,15 @@ struct _JNIEnv;
 using JNIEnv = _JNIEnv;
 class _jobject;
 using jobject = _jobject*;
+struct AInputEvent;
+
+/// \brief Event fired by ezAndroidUtils::s_InputEvent.
+/// Event listeners should inspect m_pEvent and set m_iReturn to 1 if they handled the event.
```

Comment:
Does m_iReturn have any other meaning? If it's just about handled vs. not-handled, then it should be a bool.

Reply:
Looked at the docs "Return 1 if you have handled the event, 0 for any default dispatching.". So yes, should be fine to change it to just a bool.